code/src/components/ThoughtList.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./happythoughts.css";
+
+export const ThoughtList = (props) => {
+  const { _id, message, hearts } = props.thought;
+
+  const handleHeartClick = () => {
+    fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts${_id}/like`, {
```

Comment:
`https://happy-thoughts-technigo.herokuapp.com/thoughts/${_id}/like` you forgot `/` before `${_id}`

Reply:
fixed

Reply:
Done! Tnanks!